explorer_frontend/src/features/tutorial/TutorialsPanel.tsx

```diff
@@ -0,0 +1,195 @@
+import { COLORS, Card, HeadingLarge, StyledBody } from "@nilfoundation/ui-kit";
+import { useUnit } from "effector-react";
+import { useStyletron } from "styletron-react";
+import {
+  changeActiveTab,
+  openTutorialText,
+  setSelectedTutorial,
+} from "../../pages/tutorials/model";
+import { getMobileStyles } from "../../styleHelpers";
+import { tutorialWithUrlStringRoute } from "../routing/routes/tutorialRoute";
+import { useMobile } from "../shared/hooks/useMobile";
+import { $completedTutorials, TutorialLevel } from "./model";
+
+const TutorialContainer = ({ tutorial }) => {
+  const [isMobile] = useMobile();
+  const [css] = useStyletron();
+
+  const tutorialColor = (() => {
```

Comment:
it's only advice: but you could make table where key would be enum and value is color